packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -269,7 +270,7 @@ export default async args => {
     function getClientComponentPlugins() {
         return [
             isTypescript ? widgetTyping({ sourceDir: join(sourcePath, "src") }) : null,
-            command([`npm run format --prefix ${sourcePath}`]),
+            command(() => execSync(`npm run format --prefix ${sourcePath}`, { stdio: "inherit" })),
```

Comment:
```suggestion
            platform === "native" ? command(() => execSync(`npm run format --prefix ${sourcePath}`, { stdio: "inherit" })) : null,
```

Reply:
This command is being execute twice if an editorConfig.ts or editorPreview.ts is present